data/helpers.d/package

```diff
@@ -35,15 +35,15 @@ ynh_apt() {
 #
 # usage: ynh_package_update
 ynh_package_update() {
-    ynh_apt update
+    ynh_apt --no-remove update
```

Comment:
Does this make sense ? I'm not sure there's any risk to remove a package during an apt-get update (it only refreshes the repository database thing)